src/parser/druid/restoration/modules/items/azeritetraits/EarlyHarvest.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import { formatPercentage, formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+
+// Limitations: does not handle innervate already being on cooldown when the fight begins 
+// Testing log w/ cd reduction casts https://www.warcraftlogs.com/reports/T6NCZ79yzK4D32x8#fight=1&type=healing&source=4&options=8
+
+const INNERVATE_COOLDOWN = 120000;
+const INNERVATE_DURATION = 12000;
+const COOLDOWN_REDUCTION = 1000;
+
+class EarlyHarvest extends Analyzer{
+  currentInnervateCooldown = 0;
+  lastInnervateTimestamp = 0
+  innervateCooldownReduced = 0;
+  healingFromWildGrowthExpiration = 0;
+  numEarlyHarvestHeals = 0
+  numEarlyHarvestCooldownReductions = 0
+
+  constructor(...args){
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.EARLY_HARVEST_TRAIT.id);
+  }
+
+  // Save innervate timestamp/cooldown to check it's actually on cd when cd would be reduced by early harvest 
+  on_byPlayer_cast(event) {
+    if(SPELLS.INNERVATE.id === event.ability.guid && event.sourceId === event.targetId) {
+      this.currentInnervateCooldown = INNERVATE_COOLDOWN + INNERVATE_DURATION;
+      this.lastInnervateTimestamp = event.timestamp;
+    }
+  }
```

Comment:
SpellUsable can handle this.

Reply:
addressed in commit ce6a2ba849b30945139559efb9490f713436f4aa